admin/app/jobs/R2PagePressJob.scala

```diff
@@ -0,0 +1,113 @@
+package jobs
+
+import com.amazonaws.regions.{Region, Regions}
+import com.amazonaws.services.sqs.AmazonSQSAsyncClient
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest
+import common._
+import conf.Configuration
+import conf.switches.Switches.R2PagePressServiceSwitch
+import org.jsoup.Jsoup
+import pagepresser.{PollsHtmlCleaner, BasicHtmlCleaner}
+import play.api.libs.json.Json
+import play.api.libs.ws.WS
+import services.{PagePresses, R2Archive}
+import play.api.Play.current
+
+object R2PagePressJob extends ExecutionContexts with Logging {
+  val waitTimeSeconds = Configuration.r2Press.pressQueueWaitTimeInSeconds
+  val maxMessages = Configuration.r2Press.pressQueueMaxMessages
+
+  val cleaners = Seq(BasicHtmlCleaner, PollsHtmlCleaner)
+
+  val queue: TextMessageQueue[SNSNotification] = (Configuration.r2Press.sqsQueueUrl map { queueUrl =>
+    val credentials = Configuration.aws.mandatoryCredentials
+
+    TextMessageQueue[SNSNotification](
+      new AmazonSQSAsyncClient(credentials).withRegion(Region.getRegion(Regions.EU_WEST_1)),
+      queueUrl
+    )
+  }) getOrElse {
```

Comment:
You could just `getOrElse` the configuration url directly (and inline if you wanted), makes it much easier to read!